os/sdk-modifying-coreos.md

```diff
@@ -84,37 +106,67 @@ tooling.
 ./chromite/bin/cros_sdk
 ```
 
-**WARNING:** If you ever need to delete the SDK chroot use
-`./chromite/bin/cros_sdk --delete`. Otherwise, you will delete `/dev`
-entries that are bind mounted into the chroot.
+**WARNING:** To delete the SDK chroot, use `./chromite/bin/cros_sdk --delete`. Otherwise, you will delete `/dev` entries that are `bind`-mounted into the chroot.
 
-Set up the "core" user's password.
+Set up user `core`'s password:
 
 ```sh
 ./set_shared_user_password.sh
 ```
 
-Setup a board root filesystem for the amd64-usr target in /build/amd64-usr:
+#### Selecting the architecture to build
+
+##### 64 bit AMD: The `amd64-usr` target
+
+The `--board` option can be set one of a few known target architectures, or system "boards" to build for a given CPU.
+
+To create a root filesystem for the `amd64-usr` target beneath the directory `/build/amd64-usr/`:
 
 ```sh
 ./setup_board --default --board=amd64-usr
 ```
 
+##### 64 bit ARM: The `arm64-usr` target
+
+Similarly, use `arm64-usr` for the cross-compiled ARM target. If switching
+between different targets in a single SDK, you can add the `--board=` option
+to the `build_packages`, `build_image`, and other similar commands to select the given target architecture and path.
+
+```sh
+./setup_board --default --board=arm64-usr
+```
+
+#### Compile and link system binaries
+
 Build all of the target binary packages:
 
 ```sh
-./build_packages
+./build_packages [ --board=arm64-usr ]
```

Comment:
I'd say leave off the --board arg to keep the examples copy/pasteable because people will copy/paste anyway. You can add a note to the text above that `--board=` may be needed.